invokeai/app/invocations/compel.py

```diff
@@ -193,11 +192,9 @@ def _lora_loader() -> Iterator[Tuple[LoRAModelRaw, float]]:
         for trigger in extract_ti_triggers_from_prompt(prompt):
             name = trigger[1:-1]
             try:
-                ti_model = context.models.load_by_attrs(
-                    model_name=name, base_model=text_encoder_info.config.base, model_type=ModelType.TextualInversion
-                ).model
-                assert isinstance(ti_model, TextualInversionModelRaw)
-                ti_list.append((name, ti_model))
+                loaded_model = context.models.load(key=name).model
```

Comment:
I would rename `name` to `key` on line 193 just to make it clear that this is no longer using the embedding name